src/components/settings/EditOrganizationInformationsDialog.tsx

```diff
@@ -14,20 +14,19 @@ import {
   useUpdateOrganizationInformationsMutation,
 } from '~/generated/graphql'
 import { theme } from '~/styles'
-import CountryCodes from '~/public/countryCodes.json'
+import { CountryCodes } from '~/countryCodes'
 import { addToast } from '~/core/apolloClient'
 
 const FILE_MAX_SIZE = 800000
 
-const countryData: { value: string; label: string }[] = Object.keys(CountryCodes).map(
-  (countryKey) => {
-    return {
-      value: countryKey,
-      // @ts-ignore
-      label: CountryCodes[countryKey],
-    }
+const countryData: { value: string; label: string }[] = (
```

Comment:
This should be the `BasicComboBoxData` type

Reply:
Indeed will look into it